ViewModel/TableWithPlot/TableWithPlotVM.cs

```diff
@@ -112,7 +126,19 @@ public XElement ToXML()
             ele.Add(ComputeComponentVM.ToXML());
             return ele;
         }
-
+        private void LoadFromXML(XElement ele)
+        {
+            _reverseXAxis = bool.Parse(ele.Attribute("ReverseXAxis").Value);
+            var elements = ele.Descendants();
+            XElement computeCompElement = elements.First();
+            string componentType = computeCompElement.Name.ToString();
+            switch (componentType)
```

Comment:
A little strange to have a switch with only one option. Are we expecting more down the line? If not, this should probably just be an "if".

Reply:
Fixed. used to have multiple cases. In your setup we don't.